stor/tests/test_dx.py

```diff
@@ -1386,6 +1502,17 @@ def test_dx_canonical_to_dx_file(self):
 
 
 class TestCopyTree(DXTestCase):
+    def test_empty_path(self):
+        self.setup_temporary_project()
+        dx_p = DXPath('dx://')
+        dest = DXPath('dx://' + self.project)
+        with pytest.raises(exceptions.NotFoundError, match='No project or directory was found'):
+            dx_p.copytree(dest)
+        with pytest.raises(NotImplementedError):
+            dx_p._clonetree(dest)
```

Comment:
seems kinda unnecessary - alway why do d`NotImplementedError` here?

Reply:
I didn't implement specifically. Just canonical_project is not implemented, and this hits that. Just included in test to make it explicit